default.nix

```diff
@@ -310,15 +310,16 @@ in rec {
         mv $NIX_BUILD_TOP/temp "$PWD/deps/${pname}"
         cd $PWD
 
-        ln -s ${deps}/deps/${pname}/node_modules "deps/${pname}/node_modules"
-
+        if [ -d ${deps}/deps/${pname}/node_modules ]; then
+          cp -r ${deps}/deps/${pname}/node_modules "deps/${pname}/node_modules"
+          chmod -R +w "deps/${pname}/node_modules"
+        fi
         cp -r $node_modules node_modules
         chmod -R +w node_modules
 
         ${linkDirFunction}
 
         linkDirToDirLinks "$(dirname node_modules/${pname})"
-        ln -s "deps/${pname}" "node_modules/${pname}"
```

Comment:
`node_modules` already contains a `${pname}` link to `../deps/${pname}` so what this is actually doing is creating a link to `deps/${pname}` inside of `deps/${pname}` so I assume it wasn't doing anything particularly useful.

Even if `node_modules` didn't already contain the link, it would still be wrong since the link should have been to `../deps/${pname}`.